requirements.txt

```diff
@@ -10,6 +10,7 @@ pandas>=1.0.1
 pvlib==0.7.1
 smbus2==0.3.0
 Flask-Cors==3.0.8
+tzlocal>=2.0.0
```

Comment:
Missing pytz here unless you're not using it. It was imported in another file.

Reply:
pytz is included in python stdlib now, apparently. When i tried installing it, it told me it already was installed so i didn't inculde it.

Reply:
oh i see, that's cool!